backend/src/main/java/com/google/models/MetricValue.java

```diff
@@ -27,6 +29,10 @@ public MetricValue(int value) {
     this.value = value;
   }
 
+  public MetricValue(long value) {
```

Comment:
btw we don't have a MetricValue Test class is that correct?

Reply:
we do have a `MetricValueTest.java` class! but there's only a couple of tests in that class rn...